test/unit/mdc-drawer/mdc-persistent-drawer.js

```diff
@@ -0,0 +1,284 @@
+/**
```

Comment:
This should be named `mdc-persistent-drawer.test.js`. It actually might not be getting run otherwise...

Also FYI chai has `assert.{isTrue,isFalse}` which is a strict boolean test rather than just a test for truthiness / falsiness.